frontend/public/src/components/CourseInfoBox.js

```diff
@@ -0,0 +1,143 @@
+                  alt="Expected Length and Effort"
+                />
+              </div>
+              <div className="enrollment-info-text">
+                {course.page.length}
+                {run && run.is_self_paced ? (
+                  <span className="badge badge-pacing">SELF-PACED</span>
+                ) : null}
+                {course.page.effort ? (
+                  <>
+                    <div className="enrollment-effort">
+                      {course.page.effort}
+                    </div>
+                  </>
+                ) : null}
+              </div>
+            </div>
+          ) : null}
+          <div className="row d-flex align-items-center">
+            <div className="enrollment-info-icon">
+              <img src="/static/images/products/cost.png" alt="Cost" />
+            </div>
+            <div className="enrollment-info-text font-weight-bold">Free</div>
+          </div>
+          <div className="row d-flex align-items-top">
+            <div className="enrollment-info-icon">
+              <img
+                src="/static/images/products/certificate.png"
+                alt="Certificate Track Information"
+              />
+            </div>
+            <div className="enrollment-info-text">
+              {product ? (
+                <>
+                  Certificate track: $
+                  {product.price.toLocaleString("en-us", {
+                    style:    "currency",
+                    currency: "en-US"
+                  })}
+                  {run.upgrade_deadline ? (
+                    <>
+                      <div className="text-danger">
+                        Payment deadline:{" "}
+                        {formatPrettyDate(moment(run.upgrade_deadline))}
+                      </div>
+                    </>
+                  ) : null}
+                  <div>
+                    <a target="_blank" rel="noreferrer" href="#">
+                      What's the certificate track?
+                    </a>
+                  </div>
+                  {course.page.financial_assistance_form_url ? (
+                    <div>
+                      <a
+                        target="_blank"
+                        rel="noreferrer"
+                        href={course.page.financial_assistance_form_url}
+                      >
+                        Financial assistance available
+                      </a>
+                    </div>
+                  ) : null}
+                </>
+              ) : (
+                "No certificate available."
+              )}
+            </div>
+          </div>
+        </div>
+        {course && course.programs && course.programs.length > 0 ? (
+          <div className="program-info-box">
+            <strong>
+              Part of the following program
+              {course.programs.length === 1 ? null : "s"}
+            </strong>
+
+            <ul>
+              {course.programs.map(elem => (
```

Comment:
This seems to list one program as two:
<img width="405" alt="Screen Shot 2023-08-21 at 3 47 21 PM" src="https://github.com/mitodl/mitxonline/assets/7574259/277cec92-5409-4361-9b19-7f7c95e5982e">

Reply:
I can't reproduce this - for a course that's just in one program, or is in two, it lists the proper programs out. (Do you perhaps have a program named "program" in your copy somewhere?)